src/coreclr/inc/readytorun.h

```diff
@@ -19,10 +19,10 @@
 //  src/coreclr/nativeaot/Runtime/inc/ModuleHeaders.h
 // If you update this, ensure you run `git grep MINIMUM_READYTORUN_MAJOR_VERSION`
 // and handle pending work.
-#define READYTORUN_MAJOR_VERSION 12
+#define READYTORUN_MAJOR_VERSION 13
```

Comment:
This should be update in the two other files listed above as well.

Reply:
Done, and checked there's currently no pending work for `MINIMUM_READYTORUN_MAJOR_VERSION` bump.